build.cmd

```diff
@@ -17,19 +27,29 @@ if not defined VisualStudioVersion (
     exit /b 1
 )
 
-:CheckNative
-
+:Build
+:: Restore the Tools directory
 call %~dp0init-tools.cmd
 
+:: Call the builds
+if "%__buildSpec%"=="native" goto :BuildNative
```

Comment:
when you eliminate both this can just be changed to if "%__buildSpec%"=="managed goto :BuildManaged